helm/values.yaml

```diff
@@ -31,6 +31,9 @@ serviceAccount:
   # If not set and create is true, a name is generated using the fullname template
   name: ""
 
+rbac:
+  legacy: true
```

Comment:
how about we use a different name than `.Values.rbac.legacy`? perhaps something like `rbac.fullAccess`/`rbac.restrictive/rbac.permissive`

since kro is still in alpha, the term "legacy" implies the current approach is outdated. we could use a slightly more neutral terminology to frame these as two valid configuration options during the alpha phase rather than suggesting one is deprecated.

Otherwise, the implementation looks clean :)

Reply:
Yeah, I felt already when creating this that the naming here isn't great. I'll come up with something less value-laden!

Reply:
Revamped this to be `rbac.mode: unrestricted` (current behavior) or `rbac.mode: aggregation` (new, recommended, behavior) instead; that's more descriptive and less value-laden.